main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -32,6 +32,10 @@ import java.nio.file.Paths
   */
 object Main {
 
+  private val EXIT_SUCCESS = 0
+  private val EXIT_FAILURE = 1
```

Comment:
I dont like these. Let us just inline them. Everyone should know that 0 is success :)